src/style.css

```
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

```suggestion
@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Book.ttf);
}

@font-face {
  font-family: Avenir;
  font-weight: bold;
  src: url(fonts/Avenir-Heavy.ttf);
}
```
here you import different weights for same font-family - Avenir